src/index.js

```diff
@@ -18,6 +18,20 @@ module.exports = function(source) {
             writeToDisk: false
         });
 
+        this.clearDependencies();
```

Comment:
With further testing, `clearDependencies()` is probably too aggressive. One regression is that `styles` in the `marko` file no longer reload. Unfortunately, I'm not sure at the moment whether there is more fine-grained control in Webpack to control changes in dependencies other than to wipe and re-add them all. I will test further with `clearDependencies()` removed.